tests/test_hyperkube_config.py

```diff
@@ -0,0 +1,64 @@
+import os
+import sys
+import requests
+import unittest
+
+from click.testing import CliRunner
+from importlib.util import spec_from_loader, module_from_spec
+from importlib.machinery import SourceFileLoader
+
+
+TEST_DIR = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(TEST_DIR, '..'))
+spec = spec_from_loader(
+    "kubectl-hyperkube",
+    SourceFileLoader("kubectl-hyperkube", "./cli/kubectl-hyperkube"))
+kubectl_hyperkube = module_from_spec(spec)
+spec.loader.exec_module(kubectl_hyperkube)
+
+
+class TestHyperKubeApplication(unittest.TestCase):
+    """Test hyperkube application config and functions"""
+
+    def test_help(self):
+        runner = CliRunner()
+        result = runner.invoke(
+            kubectl_hyperkube.cli,
+            ['--config', 'config_test_hyperkube.yaml', '--help'])
+        assert result.exit_code == 0
+
+    def test_load_app_config(self):
+        config = kubectl_hyperkube._load_config(
+            'tests/config_test_hyperkube.yaml')
+        assert config['stage'] == 'dev'
+        assert config['x_api_key'] == (
+            '1234567891abcfedrtas123454321Z1234567891')
+        assert config['aws_profile'] == 'default'
+        assert config['region'] == 'us-west-2'
+
+    def test_signature_generation_for_iam(self):
```

Comment:
Can you describe what this test is doing?  It looks like you are not mocking boto and making direct requests to AWS.

Reply:
Ill add description! Yeah, it is just testing that the class override I put in place to allow the use of aws profile works with aws_requests_auth module (which doesn't support aws profile directly).